README.md

```diff
@@ -94,7 +94,9 @@ Following inputs can be used as `step.with` keys:
 
 | Name             | Type    | Default                            | Description                                   |
 |------------------|---------|------------------------------------|-----------------------------------------------|
+| `scan-type`      | String  | `image`                            | Scan type, e.g. `image` or `fs` (default: `image`)|
```

Comment:
I realized we have a default column. In that case
```suggestion
| `scan-type`      | String  | `image`                            | Scan type, e.g. `image` or `fs`|
```

Reply:
Done